macos/Sources/Features/Terminal/TerminalView.swift

```diff
@@ -56,23 +56,18 @@ struct TerminalView<ViewModel: TerminalViewModel>: View {
 
     // The title for our window
     private var title: String {
-        var title = "👻"
-
-        if let surfaceTitle = surfaceTitle {
-            if (surfaceTitle.count > 0) {
-                title = surfaceTitle
-            }
+        if let surfaceTitle = surfaceTitle, !surfaceTitle.isEmpty {
```

Comment:
Note that you can shorten the first `let` assignment here:

```suggestion
        if let surfaceTitle, !surfaceTitle.isEmpty {
```